packages/wallets/src/features/cashier/modules/TransactionStatus/TransactionStatus.scss

```diff
@@ -34,6 +34,7 @@
 
     @include mobile {
         width: 100%;
+        right: 0;
         position: relative;
```

Comment:
I think the way you are handling the positioning in css is wrong my friend. 
What you want todo is to set the parent as `position: relative` and the child as `position: absolute`. This will position the child **in relative** to the parent. 

An article explaining about it: https://css-tricks.com/absolute-positioning-inside-relative-positioning/

Here's an example:
`.parent {
        position: relative;
}

.child {
    position: absolute;
    top: 3px;
    right: 3px;
}
`

Reply:
cc: @nijil-deriv